app.ts

```diff
@@ -26,6 +31,6 @@ const listener = app.listen(port, () => {
   console.log(`Server running on http://localhost:${port}`);
 });
 
-connect(process.env.CONNECTION_STRING ?? "");
+connect(process.env.MONGO_CONNECTION_STRING ?? "");
```

Comment:
This variable change MONGO_CONNECTION_STRING  is not in the env.demo, if this change is not needed let's just keep using CONNECTION_STRING

Reply:
I changed it just because it was not so easy to grasp... Like its just a general connection string... I think it would be better to be more specific... I'll update the env.demo if required

Reply:
change the demo file then